globus_cli/commands/logout.py

```diff
@@ -0,0 +1,43 @@
+import click
+
+from globus_cli.safeio import safeprint
+from globus_cli.parsing import common_options
+from globus_cli.config import (
+    AUTH_RT_OPTNAME, TRANSFER_RT_OPTNAME,
+    AUTH_AT_OPTNAME, TRANSFER_AT_OPTNAME,
+    AUTH_AT_EXPIRES_OPTNAME, TRANSFER_AT_EXPIRES_OPTNAME,
+    WHOAMI_ID_OPTNAME, WHOAMI_USERNAME_OPTNAME,
+    WHOAMI_EMAIL_OPTNAME, WHOAMI_NAME_OPTNAME,
+    internal_auth_client, remove_option, lookup_option)
+
+
+@click.command('logout',
+               short_help=('Logout of Globus, revoking credentials for '
+                           'the Globus CLI'),
+               help=('Remove credentials for the Globus CLI. '
+                     'Effectively stops any use of your CLI credentials as '
+                     'well'))
```

Comment:
Technically we're not logging the user out of Globus (at least not all of Globus, since they could very well still have an active Auth session in their browser). We're probably better off just saying "Log out of the Globus CLI" or something.

Also, I'm wondering if we shouldn't avoid using "credentials" here, since it is a technical term in oauth2 and we're not really doing anything with credentials? Perhaps say we're invalidating/revoking tokens, or maybe not even give a description of what we're actually doing under the hood at all?